Makefile

```diff
@@ -5,4 +5,14 @@ SOURCE   = https://github.com/ofiwg/$(NAME)/archive/v$(VERSION).tar.$(SRC_EXT)
 sle12_REPOS := --repo https://download.opensuse.org/repositories/science:/HPC/openSUSE_Leap_42.3/
 sl42_REPOS  := --repo https://download.opensuse.org/repositories/science:/HPC/openSUSE_Leap_42.3/
 
+ifneq ($(DAOS_STACK_REPO_SUPPORT),"")
+ifneq ($(DAOS_STACK_REPO_UBUNTU_18_04_LIST),"")
+ubuntu1804_REPOS := $(shell curl $(DAOS_STACK_REPO_SUPPORT)$(DAOS_STACK_REPO_UBUNTU_18_04_LIST))
+# Additional repos can be added but must be separated by a | character.
+endif
+# Need to figure out how to support multiple keys, such as for IPMCTL
+ifneq ($(DAOS_STACK_REPO_PUB_KEY),"")
+ubuntu1804_KEY = /tmp/daos-stack-public.key
```

Comment:
Wouldn't this be better to fetch at run-time rather than hard-coding into the docker image with no way to update it but to manually remove the docker image?

Like we do for SUSE distros for example.

Reply:
I have not tested pulling the key from the workspace yet, or other way of getting the key contents into a makefile string.